src/libs/ReportUtils.js

```diff
@@ -961,6 +961,37 @@ function getTransactionReportName(reportAction) {
     });
 }
 
+/**
+ * Get money request message for a IOU report
+ *
+ * @param {Object} report
+ * @param {Object} reportAction
+ * @returns  {String}
+ */
+function getMoneyRequestReportActionMessage(report, reportAction) {
+    let formattedAmount = null;
+    let isHavingOustandingIOU = null;
+    const totalAmount = getMoneyRequestTotal(report);
+    const payerName = isExpenseReport(report) ? getPolicyName(report) : getDisplayNameForParticipant(report.managerEmail, true);
+
+    if (totalAmount > 0) {
+        formattedAmount = CurrencyUtils.convertToDisplayString(totalAmount, report.currency);
+        isHavingOustandingIOU = report.hasOutstandingIOU;
+    } else {
+        const reportActionMessage = lodashGet(reportAction, 'message[0].html', '');
+
+        // The totalAmount is 0 after sign in/sign up because OpenApp API won't return iouReports if it's paid, so we need to retrieve it from the reportActionMessage
+        // reportActionMessage is in the format of either "payer@mail.com paid $1.00" or "paid $1.00"
+        formattedAmount = _.last(reportActionMessage.split(' '));
+        isHavingOustandingIOU = Boolean(reportActionMessage.match(/ paid /));
```

Comment:
Shouldnt this be negated? basically if paid is included then it does not have outstanding iou
```suggestion
        hasOutstandingIOU = !Boolean(reportActionMessage.match(/ paid /));
```